### PR TITLE
P0 screens: Login page

### PR DESCRIPTION
**Pull request by pritam-kr**
### First PR
- [Create a route for login page](https://github.com/pritam-kr/like/commit/3951ca4e8bf45069b9d4aff968e830b7eb03b658)
- [Feat: Created login Page](https://github.com/pritam-kr/like/commit/6227cce66adb0bad4182d60afae9420282b7a9ab)
newwww

### CODE REVIEW

**Review by himadri2110 — APPROVED** [review]
Everything looks good.
Responsiveness looks great.
Just one suggestion: You should not put a `required` attribute to the Remember Me checkbox. The user should be given a choice whether they want to be remembered or not.

**Comment by pritam-kr** [comment]
> 

Thank you. I removed the required keyword.

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,3 @@
...
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-  }
-  
-  code {
-    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-      monospace;
-  }
\ No newline at end of file
+@tailwind base;
```
Comment (horsemaker) [inline src/index.css:1]: You are using `tailwind`! Learning something new. 👌

**Review by horsemaker — APPROVED** [review]
Clean Code.
Understandable code.
Will get to learn Tailwind from you! 👍

**Comment by pritam-kr** [comment]
Added more three page
- Signup page
- 404 page
- logout page

 
Screenshot (122)

Screenshot (121)

Screenshot (123)

**Pull request merged**